sirepo/template/code_variable.py

```diff
@@ -63,6 +66,29 @@ def eval_var_with_assert(self, expr):
         assert not err, err
         return float(v)
 
+    def get_application_data(self, args, schema, ignore_array_values=False):
```

Comment:
I don't like the switch from `data` to `args`. It should validate the arguments before using them.